Add note about custom distDir to standalone build docs (#48592)
vercel/next.js: docs/advanced-features/output-file-tracing.md

Additionally, a minimal `server.js` file is also output which can be used instead of `next start`. This minimal server does not copy the `public` or `.next/static` folders by default as these should ideally be handled by a CDN instead, although these folders can be copied to the `standalone/public` and `standalone/.next/static` folders manually, after which `server.js` file will serve these automatically.

> > **Note**: If a custom [`distDir`](docs/api-reference/next.config.js/setting-a-custom-build-directory.md) has been set, the standalone build can be found at `<distDir>/standalone`. In addition, the static files have to be copied to `standalone/<distDir>/static` if they should be served by the integrated server.

Reviewer: ```suggestion
> **Note**: If [`distDir`](/docs/api-reference/next.config.js/setting-a-custom-build-directory.md) is configured, the standalone build can be found at `<distDir>/standalone`. Additionally, static files should be copied to `standalone/<distDir>/static` if they should be served by the integrated server.
```
